tests/layer_tests/tensorflow_tests/test_tf_Add.py

```diff
@@ -30,11 +31,9 @@ def create_add_placeholder_const_net(self, x_shape, y_shape, ir_version):
         with tf.compat.v1.Session() as sess:
             tf_x_shape = x_shape.copy()
             tf_y_shape = y_shape.copy()
-            # reshaping
-            if len(tf_x_shape) >= 3:
-                tf_x_shape.append(tf_x_shape.pop(1))
-            if len(tf_y_shape) >= 3:
-                tf_y_shape.append(tf_y_shape.pop(1))
+
+            tf_x_shape = permute_nchw_to_nhwc(tf_x_shape, use_new_frontend)
```

Comment:
looks like it would be better to introduce some class aka namespace with overloaded operations according to `use_new_frontend` value. It should decrease number of refactoring

Reply:
@vurusovs The task was pass the flag in the current infrastructure. So I suggest to create a new ticket to refactor these tests and do it in the next PR
I want to enable --use_new_frontend flag in precommit as soon as possible

Reply:
Let's consider it as idea for improvement, so I resolve discussion to unblock merge